packages/plugin/src/Services/DiagnosticsService.php

```diff
@@ -170,306 +237,273 @@ public function getFreeformStats()
-
-        $integrationList = [];
-        foreach ($integrations as $integration) {
-            $integrationList[] = $integration->name;
-        }
-
-        foreach ($webhooks as $webhook) {
-            $integrationList[] = $webhook->name;
-        }
-
-        $formTypes = [];
-        foreach ($freeform->formTypes->getTypes(false) as $formType) {
-            $formTypes[] = $formType['name'];
-        }
-
-        return [
+        $diagnosticItems = [
             new DiagnosticItem(
-                '<b>{{ value }}</b> Forms',
+                'Forms: <b>{{ value }}</b>',
                 $statistics->totals->forms
             ),
             new DiagnosticItem(
-                '<b>{{ value }}</b> Fields',
+                'Fields: <b>{{ value }}</b>',
                 $statistics->totals->fields
             ),
             new DiagnosticItem(
-                '<b>{{ value }}</b> Submissions',
+                'Favorites Fields: <b>{{ value }}</b>',
+                $statistics->totals->favoriteFields
+            ),
+            new DiagnosticItem(
+                'Submissions: <b>{{ value }}</b>',
                 $statistics->totals->submissions
             ),
             new DiagnosticItem(
-                $isSpamFolderEnabled
-                    ? '<b>{{ value }}</b> Submissions marked as Spam'
-                    : '<b>{{ value }}</b> Submissions blocked as Spam',
+                'Spam Submissions: <b>{{ value }}</b>',
                 $statistics->totals->spam
             ),
             new DiagnosticItem(
-                '<b>{{ value }}</b> Formatting templates',
+                'Formatting templates: <b>{{ value }}</b>',
                 \count($formTemplates)
             ),
             new DiagnosticItem(
-                '<b>{{ value }}</b> Email Notification templates',
+                'Email Notification templates: <b>{{ value }}</b>',
                 \count($emailTemplates)
             ),
             new DiagnosticItem(
-                '<b>{{ value|length }}</b> API integrations{{ value|length ? ": " }}{{ value|join(", ") }}',
-                $integrationList
+                'Success templates: <b>{{ value }}</b>',
+                \count($successTemplates)
             ),
             new DiagnosticItem(
-                '<b>{{ value|length }}</b> Additional form types{{ value|length ? ": " }}{{ value|join(", ") }}',
-                $formTypes
+                'Integrations: <b>{{ value }}</b>',
+                \count($integrations)
             ),
         ];
+
+        // Check if Freeform Pro is enabled, then add the Form types item
+        if ($freeform->isPro()) {
+            $diagnosticItems[] = new DiagnosticItem(
+                'Form types: <b>{{ value|length }}</b>',
+                \count($formTypes)
+            );
+        }
+
+        return $diagnosticItems;
     }
 
-    public function getFreeformChecks()
+    public function getFreeformIntegrations()
```

Comment:
Lets strive to always define return types of functions.

```suggestion
    /**
     * @return DiagnosticItem[]
     */
    public function getFreeformIntegrations(): array
```

In this case, we would also need a @DocType since the `array` return type doesn't encompass the type of elements inside the array. Otherwise the @DocType is unnecessary.